dnf5/commands/check-upgrade/check-upgrade.cpp

```diff
@@ -113,8 +113,10 @@ void CheckUpgradeCommand::run() {
     // filter by provided specs, for `check-upgrade <pkg1> <pkg2> ...`
     if (!pkg_specs.empty()) {
         upgrades_query = libdnf5::rpm::PackageQuery(ctx.base, libdnf5::sack::ExcludeFlags::APPLY_EXCLUDES, true);
-        libdnf5::ResolveSpecSettings settings{
-            .with_nevra = true, .with_provides = false, .with_filenames = false, .with_binaries = false};
+        libdnf5::ResolveSpecSettings settings;
+        settings.set_with_nevra(true);
+        settings.set_with_filenames(false);
```

Comment:
Isn't `set_with_provides(false)` missing here?

Reply:
Oh right, thanks for catching those!